### PR TITLE
:construction_worker: Add golangci-lint

### PR DESCRIPTION
**Pull request by rinx**
@kpango Could you add a "github-token" to secrets of this repository? ( I couldn't access to that page.)
ref: https://github.com/reviewdog/action-misspell

### CODE REVIEW

**Comment by kpango** [comment]
@rinx thank you for your contribution. I'm considering about using golangci-lint it also include misspell, do you mind if we start using [this](https://github.com/marketplace/actions/run-golangci-lint-with-reviewdog)

**Comment by rinx** [comment]
@kpango I didn't know about it! I also think it is better to use it.
I'll revise this PR later. Thx. :smiley:

**Comment by kpango** [comment]
LGTM

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  golangci-lint:
+    name: runner / golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v1
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```
[the remark this answers is not in the document]
Comment (kpango) [reply .github/workflows/reviewdog.yml]: @rinx how do you think?

[the remark this answers is not in the document]
Comment (rinx) [reply .github/workflows/reviewdog.yml]: Do you mean this?
https://help.github.com/en/articles/virtual-environments-for-github-actions#github_token-secret
Okay. I'm gonna fix it. :+1:

[the remark this answers is not in the document]
Comment (kpango) [reply .github/workflows/reviewdog.yml]: yes I mean that.
Thx!

**Comment by kpango** [comment]
LGTM

**Pull request merged**